certification/internal/policy/operator/scorecard_basic_check_spec.go

```diff
@@ -47,6 +50,13 @@ func (p *ScorecardBasicSpecCheck) Metadata() certification.Metadata {
 }
 
 func (p *ScorecardBasicSpecCheck) Help() certification.HelpText {
+	if p.fatalError {
+		return certification.HelpText{
+			Message: "There was a fatal error while running operator-sdk scorecard tests. " +
+				"Please see the preflight log for details. If necessary, set logging to be more verbose.",
+			Suggestion: "Set wait time to a higher value if this is a consistent error.",
```

Comment:
Do we want to suggest this? This just carries the implication that you'll hit this in timeout cases only. Up to you, feel free to resolve.

Reply:
This was literally just a POC. There needs to be a way to better determine what actual error occurred. Or it needs to be more generic.